build/configs/rtl8730e/cpp_loadable/defconfig

```diff
@@ -1234,7 +1271,7 @@ CONFIG_ELF_CACHE_BLOCKS_COUNT=60
 CONFIG_BINFMT_CONSTRUCTORS=y
```

Comment:
If you want to enable this, you should enable SCHED_STARTHOOK. BINFMT_CONSTRUCTORS depends on SCHED_STARTHOOK.

Reply:
Also, please enable CONFIG_HAVE_CXXINITIALIZE in cxxtest.

Reply:
Updated, thankyou